src/client/configuration.proto

```diff
@@ -16,19 +16,22 @@ message Plugin {
 }
 
 message Configuration {
-  required string path         = 1;
-  optional string version      = 2;
+  required string path            = 1;
+  optional string version         = 2;
 
-  optional uint32 send_timeout = 3 [ default = 5  ];
+  optional uint32 connect_timeout = 3 [ default = 10 ];
+  // in seconds. Zero means default.
```

Comment:
Actually, when `connect_timeout` is set to zero, we wait as long as connection gets timed out by itself, therefore, yes, this should mean "system default".